mk/sm.spec.in

```diff
@@ -100,12 +100,6 @@ exit 0
 %systemd_postun sm-mpath-root.service
 %systemd_postun xs-sm.service
 %systemd_postun storage-init.service
-if [ $1 -eq 0 ]; then
-    [ ! -d /etc/lvm/master ] || rm -Rf /etc/lvm/master || exit $?
-    cp -f /etc/lvm/lvm.conf.orig /etc/lvm/lvm.conf || exit $?
-elif [ $1 -eq 1 ]; then
-    true;
```

Comment:
I also removed this, because `%systemd_postun test.service` evaluates to:
```
rpm --eval "%systemd_preun test.service"

if [ $1 -eq 0 ] ; then 
        # Package removal, not upgrade 
        systemctl --no-reload disable test.service > /dev/null 2>&1 || : 
        systemctl stop test.service > /dev/null 2>&1 || : 
fi
```
so nothing is executed in the upgrade case, and even if there was, it always ends with a true (`:`) so there's no risk of failed update.